graph/db/test_postgres.go

```diff
@@ -0,0 +1,40 @@
+//go:build test_db_postgres && !test_db_sqlite
+
+package graphdb
+
+import (
+	"database/sql"
+	"testing"
+
+	"github.com/lightningnetwork/lnd/kvdb"
+	"github.com/lightningnetwork/lnd/sqldb"
+	"github.com/stretchr/testify/require"
+)
+
+// NewTestDB is a helper function that creates an BBolt database for testing.
```

Comment:
nit: the comment is slightly inaccurate, perhaps we could mention that it's a kv backend embedded into a postgres store, but the kv backend is bbolt by default (unless other tags are set).

Reply:
indeed! good catch - thanks. Updated